packages/private-apis/src/implementation.ts

```diff
@@ -168,10 +166,10 @@ function lock( object, privateData ) {
  * // { a: 1 }
  * ```
  *
- * @param {any} object The object to unlock the private data from.
- * @return {any} The private data bound to the object.
+ * @param {Record< symbol, WeakKey >} object The object to unlock the private data from.
```

Comment:
We could remove these from the JSDoc, the type definition below is enough.

Reply:
Done. Thanks